packages/core/test/test_utils.js

```diff
@@ -14,10 +14,11 @@ export function createComponent(Component, props) {
   const onError = sinon.spy();
   const onSubmit = sinon.spy();
   const comp = <Component onSubmit={onSubmit} onError={onError} onChange={onChange} {...props} />;
-  const { container } = render(comp);
+  const compCB = (props) => <Component onSubmit={onSubmit} onError={onError} onChange={onChange} {...props} />;
+  const { container, rerender } = render(comp);
```

Comment:
Why not provide a slight modification to the `rerender` function. Something like:
```suggestion
  const { container, rerender } = render(comp);
  
  const rerenderFunction = (props) => rerender(<Component onSubmit={onSubmit} onError={onError} onChange={onChange} {...props} />);
```